README.md

```diff
@@ -1,14 +1,144 @@
 
-## Getting started
 
-The instructions below assume you already have a Firebase project setup and configured appropriately.
+### Getting started
 
-**Note**: If you're interested in setting up the project to run in your own environment but you're not sure how to go about it, feel free to [open an issue](https://github.com/sparkletown/sparkle/issues/new) asking for assistance.
+What you will need:
 
-### Frontend
+- A free Firebase account
+- Your favorite command-line interface (e.g. Terminal or zsh on a Mac)
+- Optional: accounts with Stripe, Twillio
+- A little fairy dust
+
+---
+<!-- section 1 -->
+
+## Part One: Firebase Project Setup
+
+### Step 1: Create New Firebase Project
+
+1. Go to https://console.firebase.google.com/u/2/.
+
+2. Click _Add project_.
+  * Fill out the name field with something like `example-project`
+  * Click _Continue_
+
+
+3. Optional: enable Google Analytics.
+
+  * Click _Continue_
+  * Configure Google Analytics
+  * Click _Create project_
+  * Wait for project to be created
+  * Click _Continue_
+
+This part of the setup is complete!
+
+
+### Step 2: Configure Firebase Project Settings
+
+1. Go to https://console.firebase.google.com/ and find the `Example Project` you chose in step 1.
+
+2. From _Project Overview_, hover over the gear icon and click _Project Settings_.
+
+ * In the 'General' tab:
+
+    * Set the public-facing name.
+   * In the ['Users & Permissions'](https://console.firebase.google.com/u/2/project/`example-project`/settings/iam) tab:
+
+    * Add any other users who will need "Owner" level access to this project (e.g. a backup owner who can change things if needed, deployments).
+    * Add the users who will need "Editor" level access to this project.
+
+3. Optionally, upgrade your account. Click the gear icon next to _Project Overview_ again, and then select ['Usage and billing'](https://console.firebase.google.com/u/2/project/`example-project`/usage), then select the Blaze plan.
+
+### Step 3: Add a New Web App to the Firebase Project
+
+1. Click on _Project Overview_, then again on [_Project Settings_](https://console.firebase.google.com/u/2/project/`example-project`/settings/general).
+
+ * In the 'General' tab:
+
+    * Go to the 'Your Apps' section at the bottom.
+    * Click on the `</>` button to create a new web app
+    * Add firebase to your web app
+    * Choose an app nickname (eg. `example-project`)
+    * Tick 'Also set up Firebase Hosting for this app', leaving the dropdown that appears on the default setting
+    * Click _Register app_
+    * Wait for the _loading_ spinner to finish
+    * Click _Next_ to skip past this screen
+    * Click _Next_ to skip past this screen as well
+    * Finally, you can click _Continue to the console_ to skip past this last screen; internal deployments are handled automatically through our CircleCI workflow, so we don't need to deploy manually
```

Comment:
should we mention that non-Sparkle guys don't need to worry if they don't use Circle CI unless they want to deploy something? or how to configure everything manually? or whatever else worth mentioning here?